state/machine_linklayerdevices.go

```diff
@@ -206,18 +206,27 @@ func (m *Machine) SetLinkLayerDevices(devicesArgs ...LinkLayerDeviceArgs) (err e
 			}
 		}
 
+		// We've checked the model is alive directly, and we assert the machine is alive, we don't need to also
+		// assert the model is alive, because then the machine would be dying as well.
```

Comment:
This statement is not true. A model can be dying, but still have alive machines. They will eventually be destroyed on account of cleanups.

If we don't really care about the model life here, should we remove the one above too? The doc comment for the method does not explain *why* we check the model life. Do you know?